python/ray/rllib/models/catalog.py

```diff
@@ -14,11 +14,15 @@
 
 MODEL_CONFIGS = [
     "conv_filters",
-    "downscale_factor",
+    # TODO(rliaw) introduce specific ATARI config
+    "dim",
+    "grayscale",
+    "zero_mean",
```

Comment:
Not a problem introduced by this PR, but could we add a short comment on what each of these config options do?